community/modules/scripts/wait-for-startup/main.tf

```diff
@@ -13,22 +13,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-data "google_compute_instance" "vm_instance" {
-  name    = var.instance_name
-  zone    = var.zone
-  project = var.project_id
-}
+# data "google_compute_instance" "vm_instance" {
+#   name    = var.instance_name
+#   zone    = var.zone
+#   project = var.project_id
+# }
 
 resource "null_resource" "wait_for_startup" {
   provisioner "local-exec" {
     command = "/bin/bash ${path.module}/scripts/wait-for-startup-status.sh"
     environment = {
-      INSTANCE_NAME = var.instance_name
-      ZONE          = var.zone
-      PROJECT_ID    = var.project_id
-      TIMEOUT       = var.timeout
+      INSTANCE_NAMES = join(" ", distinct(compact(concat(var.instance_names, [var.instance_name]))))
+      ZONE           = var.zone
```

Comment:
Do we assume instances are in the same zone/project?

Reply:
Yup, that's correct, since that assumption was held before this change.  I don't want to make any breaking changes in this update, but it's something we can certainly think about as a separate change.